protocol/x/accountplus/types/keys.go

```diff
@@ -2,8 +2,9 @@ package types
 
 // Module name and store keys.
 const (
-	// ModuleName defines the module name.
-	ModuleName = "accountplus"
+	// The Account module uses "acc" as its module name.
+	// KVStore keys cannot have other keys as prefixes so we prepend "dydx" to "accountplus"
+	ModuleName = "dydxaccountplus"
```

Comment:
are there any issues with the name being dydxaccountplus, and everything else with this module being named accountplus?